priv/repo/migrations/20160322222933_addnullchatprovider.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Cog.Repo.Migrations.Addnullchatprovider do
```

Comment:
Probably not a big deal, but I think we usually make these migration names snakecase.